src/PolymakeWrap.jl

```diff
@@ -96,17 +96,64 @@ include("convert.jl")
 include("sets.jl")
 # to be moved to Vectors/Matrices
 pm_Integer(b::BigInt) = new_pm_Integer(b)
-pm_Rational(num::BigInt, den::BigInt) = pm_Rational(pm_Integer(num), pm_Integer(den))
+pm_Rational(num::T, den::T) where T<:Integer = 
+    pm_Rational(pm_Integer(num), pm_Integer(den))
+pm_Rational(r::R) where R<:Rational = pm_Rational(numerator(r), denominator(r))
```

Comment:
```suggestion
pm_Rational(r::Rational) = pm_Rational(numerator(r), denominator(r))
```
is a little but shorter and avoids the type parameter. (Obviously this is nitpicking)